src/components/Staking/ShanghaiCapella.tsx

```diff
@@ -3,14 +3,16 @@ import React, { FC } from "react"
 import {
   Box,
   Flex,
+  FlexProps,
+  ListItem,
   Text,
   UnorderedList,
-  ListItem,
-  FlexProps,
 } from "@chakra-ui/react"
+import { useTranslation } from "gatsby-plugin-react-i18next"
```

Comment:
```suggestion
import { useTranslation } from "gatsby-plugin-react-i18next"
```

I don't think this ends up being used